model-optimizer/mo/utils/cli_parser.py

```diff
@@ -75,6 +75,17 @@ def __call__(self, parser, namespace, values, option_string=None):
                             ' but "{}" does not exist.'.format(self.dest, name))
```

Comment:
should we remove `CanonicalizePathCheckExistenceAction`?

Reply:
No, it is used for other options where existing file should be set